tools/hive/hadoop-2.6.0/scripts/bootstrap.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -x
+/usr/sbin/sshd
+: ${HADOOP_PREFIX:=/usr/local/hadoop}
+
+$HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
+
+rm /tmp/*.pid
+
+# installing libraries if any - (resource urls added comma separated to the ACP system variable)
+cd $HADOOP_PREFIX/share/hadoop/common ; for cp in ${ACP//,/ }; do  echo == $cp; curl -LO $cp ; done; cd -
+
+# templating of config files
+sed s/HOSTNAME/$HOSTNAME/ /usr/local/hadoop/etc/hadoop/core-site.xml.template > /usr/local/hadoop/etc/hadoop/core-site.xml
```

Comment:
It would be great to use mustache templates as we do with our other packages.

Reply:
@elezar That's a good idea. What did you have in mind? I could continue to use `sed` except with the curly braces "{{HOSTNAME}}" in the template. Or, were you thinking of using an actual mustache implementation, such as the Go library in dcos-commons?